src/test/java/DirectionsServletTest.java

```diff
@@ -41,30 +39,28 @@
  */
 @RunWith(JUnit4.class)
 public final class DirectionsServletTest {
-  private static final DirectionsClientFactory directionsClientFactory =
-      Mockito.mock(DirectionsClientFactory.class);
-  private static final DirectionsClient directionsClient = Mockito.mock(DirectionsClient.class);
-  private static final DirectionsServlet servlet = new DirectionsServlet(directionsClientFactory);
-
-  private static HttpServletRequest request;
+  private static DirectionsClientFactory directionsClientFactory;
+  private static DirectionsClient directionsClient;
```

Comment:
Any reason for them to be static?

Reply:
Nope! I'll remove the `static` keywords to keep it standardized.

Reply:
`RESOLVED`